[opentelemetry-cpp] OTLP features don't compile anymore

### Operating system

Linux

### Compiler

GCC

### Steps to reproduce the behavior

I belive this issue comes from https://github.com/microsoft/vcpkg/pull/39632. 

In that PR, opentelemetry-cpp version was updated, but OTLP was not updated. OTLP is tightly coupled to opentelemetry-cpp repository, and OTLP should have been updated with this change.

```Shell
`./vcpkg install opentelemetry-cpp[otlp-htpp]`
```


### Failure logs

Run Build Command(s): /usr/local/bin/ninja -v -v -j9 install
[1/126] cd /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg && /mnt/c/dev/vcpkg/installed/x64-linux/tools/protobuf/protoc-25.1.0 --proto_path=/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto --cpp_out=/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/profiles.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/pprofextended.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/profiles/v1experimental/profiles_service.proto
FAILED: generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/profiles.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/profiles.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/pprofextended.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/pprofextended.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.cc generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/profiles/v1experimental/profiles_service.pb.h generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/profiles/v1experimental/profiles_service.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/profiles.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/profiles.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/pprofextended.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/pprofextended.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.pb.cc /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/profiles/v1experimental/profiles_service.pb.h /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto/opentelemetry/proto/collector/profiles/v1experimental/profiles_service.pb.cc 
cd /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg && /mnt/c/dev/vcpkg/installed/x64-linux/tools/protobuf/protoc-25.1.0 --proto_path=/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto --cpp_out=/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/common/v1/common.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/resource/v1/resource.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/trace/v1/trace.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/logs/v1/logs.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/metrics/v1/metrics.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/trace/v1/trace_service.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/logs/v1/logs_service.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/metrics/v1/metrics_service.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/profiles.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/pprofextended.proto /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/collector/profiles/v1experimental/profiles_service.proto
Could not make proto path relative: /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/third_party/opentelemetry-proto/opentelemetry/proto/profiles/v1experimental/profiles.proto: No such file or directory
[2/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/common/CMakeFiles/opentelemetry_common.dir/platform/fork_unix.cc.o -MF sdk/src/common/CMakeFiles/opentelemetry_common.dir/platform/fork_unix.cc.o.d -o sdk/src/common/CMakeFiles/opentelemetry_common.dir/platform/fork_unix.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/common/platform/fork_unix.cc
[3/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/common/CMakeFiles/opentelemetry_common.dir/random.cc.o -MF sdk/src/common/CMakeFiles/opentelemetry_common.dir/random.cc.o.d -o sdk/src/common/CMakeFiles/opentelemetry_common.dir/random.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/common/random.cc
[4/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/common/CMakeFiles/opentelemetry_common.dir/base64.cc.o -MF sdk/src/common/CMakeFiles/opentelemetry_common.dir/base64.cc.o.d -o sdk/src/common/CMakeFiles/opentelemetry_common.dir/base64.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/common/base64.cc
[5/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/common/CMakeFiles/opentelemetry_common.dir/global_log_handler.cc.o -MF sdk/src/common/CMakeFiles/opentelemetry_common.dir/global_log_handler.cc.o.d -o sdk/src/common/CMakeFiles/opentelemetry_common.dir/global_log_handler.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/common/global_log_handler.cc
[6/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/common/CMakeFiles/opentelemetry_common.dir/env_variables.cc.o -MF sdk/src/common/CMakeFiles/opentelemetry_common.dir/env_variables.cc.o.d -o sdk/src/common/CMakeFiles/opentelemetry_common.dir/env_variables.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/common/env_variables.cc
[7/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer_context_factory.cc.o -MF sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer_context_factory.cc.o.d -o sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer_context_factory.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/trace/tracer_context_factory.cc
[8/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer_context.cc.o -MF sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer_context.cc.o.d -o sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer_context.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/trace/tracer_context.cc
[9/126] /usr/bin/c++ -DHAVE_ABSEIL -DOPENTELEMETRY_ABI_VERSION_NO=1 -DOPENTELEMETRY_DEPRECATED_SDK_FACTORY -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/x64-linux-dbg/generated/third_party/opentelemetry-proto -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/api/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/include -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk -I/mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/ext/include -isystem /mnt/c/dev/vcpkg/installed/x64-linux/include -fPIC -g -std=gnu++14 -Wno-error=deprecated-declarations -MD -MT sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer.cc.o -MF sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer.cc.o.d -o sdk/src/trace/CMakeFiles/opentelemetry_trace.dir/tracer.cc.o -c /mnt/c/dev/vcpkg/buildtrees/opentelemetry-cpp/src/v1.16.0-caf34f28ce.clean/sdk/src/trace/tracer.cc
ninja: build stopped: subcommand failed.

### Additional context

_No response_

--- a/ports/opentelemetry-cpp/portfile.cmake
+++ b/ports/opentelemetry-cpp/portfile.cmake
@@ -29,11 +29,11 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
 
 # opentelemetry-proto is a third party submodule and opentelemetry-cpp release did not pack it.
 if(WITH_OTLP_GRPC OR WITH_OTLP_HTTP)
-    set(OTEL_PROTO_VERSION "1.1.0")
+    set(OTEL_PROTO_VERSION "1.3.1")
     vcpkg_download_distfile(ARCHIVE
         URLS "https://github.com/open-telemetry/opentelemetry-proto/archive/v${OTEL_PROTO_VERSION}.tar.gz"
         FILENAME "opentelemetry-proto-${OTEL_PROTO_VERSION}.tar.gz"
-        SHA512 cd20991efb2d7f1bc8650fd0e124be707922b0717e429b6212390cd2c0d0afdb403c9aece196f07ae81ebed948863f4ec75c08ffbb3968795a0010d5cb34dc1b
+        SHA512 8c75e4ff79c4b5b251e0ec8ece92ec901d70ec601644505ffdd137fb728daac91fd9203e1f448500124906737d91d80f10b694977688c655418b94f61c828d06
     )
 
     vcpkg_extract_source_archive(src ARCHIVE "${ARCHIVE}")

--- a/ports/opentelemetry-cpp/vcpkg.json
+++ b/ports/opentelemetry-cpp/vcpkg.json
@@ -2,6 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
   "name": "opentelemetry-cpp",
   "version-semver": "1.16.0",
+  "port-version": 1,
   "description": [
     "OpenTelemetry is a collection of tools, APIs, and SDKs.",
     "You use it to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior."

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6602,7 +6602,7 @@
     },
     "opentelemetry-cpp": {
       "baseline": "1.16.0",
-      "port-version": 0
+      "port-version": 1
     },
     "opentracing": {
       "baseline": "1.6.0",

--- a/versions/o-/opentelemetry-cpp.json
+++ b/versions/o-/opentelemetry-cpp.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "24f0a022ab07e290283011657cac26e32a6dd648",
+      "version-semver": "1.16.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "463e6bcd28686e4433ddd353b8f0cfd82e7c577b",
       "version-semver": "1.16.0",
       "port-version": 0

